.gitlab-ci.yml

```diff
@@ -1,11 +1,16 @@
-netlify:
-  type: deploy
+image: python:3-slim-stretch
+stages:
+  - deploy
+
+build and push:
```

Comment:
this job definitely will need a `tags` section and include tags that will match one of our runners.